samples/android/src/main/java/cafe/adriel/voyager/sample/kodeinIntegration/KodeinScreen.kt

```diff
@@ -19,7 +19,7 @@ class KodeinScreen : Screen {
 
     @Composable
     override fun Content() {
-        val screenModel = rememberScreenModel<KodeinScreenModel>()
+        val screenModel = rememberScreenModel { KodeinScreenModel() }
```

Comment:
This change here is wrong. This Screen is meant for Kodein, in this case, Kodein act as a Factory for the KodeinScreenModel and the function `rememberScreenModel<T>()` is a extension function for the Kodein library